examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "31226b14-e907-4503-9918-a09163363217",
```

Comment:
<p>nitpicking:</p><p> </p><p>Might be nicer to avoid those function and to add an option for the derivatives to "expand/simplify". so that you just need <code>grad(div(p))</code> for graddiv for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2450/discussion/#comment-1758172536'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>